scalafmt-tests/src/test/resources/unit/Dialect.stat

```diff
@@ -17,11 +17,7 @@ val x: 42 = 42
 val x : A   &  B = 42
 >>>
 val x: A & B = 42
-<<< extractor
-val List(a : _ *) = 2
->>>
-val List(a: _*) = 2
-<<< trait parameters
+<<< trait parameters - supported in default dialect to produce consistent trees in Scala 2 and Scala 3.
```

Comment:
is this related?

Reply:
This is related to the dialect changes. This PR adds a requirement to add `runner.dialect = Scala3` in order to format Scala 3 syntax. Trait parameters are a new syntax in Scala 3 but they're enabled by default in Scalafmt. The purpose of the comment here was make this decision explicit, it's not accidental